app/Libraries/UKCP.php

```diff
@@ -140,4 +144,38 @@ public static function getPathForToken($tokenID, $account)
     {
         return self::TOKEN_PATH_ROOT.$account->id.'/'.$tokenID.'.json';
     }
+
+    public function getStandStatus(string $airfield): array
+    {
+        try {
+            $cached = Cache::get($this->getStandStatusCacheKey($airfield));
+            if ($cached) {
+                return $cached;
+            }
```

Comment:
I'm sure there's a way of doing this using Cache::remember?

Reply:
Trying it here! 4ec12dce